cron/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM node:20-slim
+
+RUN apt-get update && apt-get install -y \
+    wget \
+    gnupg2 \
+    lsb-release \
+    cron \
+    && \
+    echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+    apt-get update && \
+    apt-get install -y postgresql-client-16 && \
```

```suggestion
    apt-get install -y postgresql-client-17 && \
```

Ich musste das auf 17 bumpen, weil das zu der DB-Version passen muss, und die ist auf prod anscheinend 17. Allgemein mache ich das bei meinem Server so, dass ich einen cronjob auf dem Host laufen lasse, der sich per `docker exec` in den laufenden postgres-container schaltet und dort `pg_dump` ausführt, dann hat man immer dieselbe Version, aber das hat dafür sicher andere Nachteile...
Und allgemein ist das prod deployment nicht exakt gleich (z.B. heißt der DB container dort `db`), sodass der committete Stand nicht exakt passt (ist bei meinem monitoring-Stuff auch so...) Aber ich denke für den Moment passt das so.